Team8923_PowerPlay/src/main/java/org/firstinspires/ftc/team8923_PowerPlay/BaseTeleOp.java

```diff
@@ -0,0 +1,27 @@
+package org.firstinspires.ftc.team8923_PowerPlay;
+
+abstract public class BaseTeleOp extends BaseOpMode {
+
+    private boolean isSlowMode = false;
+
+    private Toggle driveSpeedToggle = new Toggle();
+
+    double driveSpeed = 1.0;
+
+    public void driveRobot(){
```

Comment:
https://google.github.io/styleguide/javaguide.html#s4.1-braces
```suggestion
    public void driveRobot() {
```